src/java/voldemort/server/VoldemortConfig.java

```diff
@@ -567,6 +572,11 @@ public VoldemortConfig(Props props) {
         this.slopPurgeJobMaxKeysScannedPerSec = props.getInt("slop.purgejob.max.keys.scanned.per.sec",
                                                              10000);
 
+        // RocksDB config
+        this.rdbDataDirectory = props.getString("rocksdb.data.dir", "/tmp/rdb_data_dir");
```

Comment:
can we derive this from VOLDEMORT_HOME or something by default, like how bdb data directory works?